lib/ansible/modules/git.py

```diff
@@ -567,6 +567,36 @@ def get_submodule_versions(git_path, module, dest, version='HEAD'):
     return submodules
 
 
+def get_submodule_config(git_path, module, dest, submodule_name, config_name):
+    config_path = 'submodule.' + submodule_name + '.' + config_name
+    cmd = [git_path, 'config', '-f', '.gitmodules', '--get', config_path]
+    (rc, out, err) = module.run_command(cmd, cwd=dest)
```

Comment:
If this fails unexpectedly, the error should be visible to the user
```suggestion
    (rc, out, err) = module.run_command(cmd, cwd=dest, check_rc=True)
```